apps/cookbook/src/app/component-status/component-status.component.ts

```diff
@@ -432,6 +421,33 @@ export class ComponentStatusComponent implements OnInit, OnDestroy {
       })
     );
   }
+
+  getPageOfIssues(url: string) {
+    const options = {
+      headers: new HttpHeaders({
+        Authorization: 'token ' + environment.oauth.githubToken1 + environment.oauth.githubToken2,
+      }),
+      observe: 'response' as const,
+    };
+    return this.http.get<Observable<any>>(url, options).pipe(
```

Comment:
The generic type specificed in `get<T>` should no be wrapped in an Observable type - that's the type that `get` returns wrapping the `T` type:
```suggestion
    return this.http.get<any>(url, options).pipe(
```